q2_annotate/busco/utils.py

```diff
@@ -283,3 +264,42 @@ def _validate_parameters(lineage_dataset, auto_lineage,
             "If 'lineage-dataset' is provided, all the parameters 'auto-lineage', "
             "'auto-lineage-euk' and 'auto-lineage-prok' must be set to False."
         )
+
+
+def _process_busco_results(add_contam_complete, results, mag_id, file_name, sample_id):
```

Comment:
Could you please order these arguments in some more logical way? Like results, sample_id, mag_id, file_name, add_contam_complete? Sorry, my ocd cannot take this 🙈